src/router.js

```diff
@@ -3,21 +3,21 @@ import twofactorauth from './twofactorauth';
 import passkeys from './passkeys';
 
 export default {
-	async fetch(request, env) {
-		return router.handle(request, env);
-	},
+  async fetch(request, env) {
+    return router.handle(request, env);
+  }
 };
 
 const router = Router();
 
 router.get('/:repo/:pr/', async (req, env) => {
-	const { repo } = req.params;
-	switch (repo) {
-		case 'twofactorauth':
-			return twofactorauth(req, env);
-		case 'passkeys':
-			return passkeys(req, env);
-	}
+  const { repo } = req.params;
+  switch (repo) {
+    case 'twofactorauth':
+      return twofactorauth(req, env);
+    case 'passkeys':
+      return passkeys(req, env);
+  }
```

Comment:
Could we switch this to check if the repository is `twofactorauth` or `passkeys` and run the same function for both (probably the 2FA one because that's the more advanced one)? They seem to have identical functionality.

Reply:
You're right, there's a lot of duplicate lines of code.
Due to differences in the directory structures and JSON schema, I opted for separate processing functions.

It would probably have been possible to create one function to handle both structures but I chose the easiest path :sweat_smile:

Reply:
Sounds good! We can always change it later!